src/registrar/management/commands/utility/extra_transition_domain_helper.py

```diff
@@ -775,52 +963,47 @@ def __init__(self, options: TransitionDomainArguments):
         # That way if we want the data located at the agency_adhoc file,
         # we can just call EnumFilenames.AGENCY_ADHOC.
-        pattern_map_params = [
-            (
-                EnumFilenames.AGENCY_ADHOC,
-                options.agency_adhoc_filename,
-                AgencyAdhoc,
-                "agencyid",
-            ),
-            (
-                EnumFilenames.DOMAIN_ADDITIONAL,
-                options.domain_additional_filename,
-                DomainAdditionalData,
-                "domainname",
-            ),
-            (
-                EnumFilenames.DOMAIN_ESCROW,
-                options.domain_escrow_filename,
-                DomainEscrow,
-                "domainname",
-            ),
-            (
-                EnumFilenames.DOMAIN_ADHOC,
-                options.domain_adhoc_filename,
-                DomainTypeAdhoc,
-                "domaintypeid",
-            ),
-            (
-                EnumFilenames.ORGANIZATION_ADHOC,
-                options.organization_adhoc_filename,
-                OrganizationAdhoc,
-                "orgid",
-            ),
-            (
-                EnumFilenames.AUTHORITY_ADHOC,
-                options.authority_adhoc_filename,
-                AuthorityAdhoc,
-                "authorityid",
-            ),
-            (
-                EnumFilenames.AUTHORITY_ADHOC,
-                options.authority_adhoc_filename,
-                AuthorityAdhoc,
-                "authorityid",
-            ),
-        ]
+        if options.pattern_map_params is None or options.pattern_map_params == []:
+            options.pattern_map_params = [
+                (
+                    EnumFilenames.AGENCY_ADHOC,
+                    options.agency_adhoc_filename,
+                    AgencyAdhoc,
+                    "agencyid",
+                ),
+                (
+                    EnumFilenames.DOMAIN_ADDITIONAL,
+                    options.domain_additional_filename,
+                    DomainAdditionalData,
+                    "domainname",
+                ),
+                (
+                    EnumFilenames.DOMAIN_ESCROW,
+                    options.domain_escrow_filename,
+                    DomainEscrow,
+                    "domainname",
+                ),
+                (
+                    EnumFilenames.DOMAIN_ADHOC,
+                    options.domain_adhoc_filename,
+                    DomainTypeAdhoc,
+                    "domaintypeid",
+                ),
+                (
+                    EnumFilenames.ORGANIZATION_ADHOC,
+                    options.organization_adhoc_filename,
+                    OrganizationAdhoc,
+                    "orgid",
+                ),
+                (
+                    EnumFilenames.AUTHORITY_ADHOC,
+                    options.authority_adhoc_filename,
+                    AuthorityAdhoc,
+                    "authorityid",
+                ),
+            ]
```

Comment:
This is just a one line modification of the old code but get doesn't see it that way, lol. This basically just specifies what files to load from and how to parse them.

This wasn't an option you could pass in before, so this change just makes it so we don't have to load all 6 files in OrganizationDataLoader and instead we can just define the two we would want to parse.